frequency-chain/frequency-xyz
# 52 about section

# Description

Build the About section content and images.

- Closes #52 

## Screenshot(s)

https://github.com/user-attachments/assets/69038ea5-277d-45db-ba28-85163205130a

## Type of change

Please delete options that are not relevant.


# How to Test?

1. Pull and run locally
2. Go to the UI
3. See that the UI looks like the mocks in all screen sizes
4. The copy is correct

# Checklist:
